Framework/PythonInterface/mantid/plots/mantidaxes.py

```diff
@@ -418,17 +439,39 @@ def replot_artist(self, artist, errorbars=False, **kwargs):
         """
         kwargs['distribution'] = not self.get_artist_normalization_state(artist)
         workspace, spec_num = self.get_artists_workspace_and_spec_num(artist)
-        self.remove_artists_if(lambda art: art == artist)
-        if kwargs.get('axis', None) == MantidAxType.BIN:
-            workspace_index = spec_num
+
+        #check if it is a sample log plot
+        if spec_num is None:
+            #error bars never make sense for sample log plots
```

Comment:
This comment is repeated further down where you set `errorbars` to false.
```suggestion
        if spec_num is None:
```